### ISSUE TITLE
Question about the implement of the lidar segmentation task

### ISSUE THREAD

**Issue by YuanxianH** [OpenDriveLab/OccNet#16]
Thank you for your awesome contributions. Recently I've been trying to apply your work in lidar segmentation tasks. My question is how you generate the lidar segmentation result from the voxel output? 

As you mentioned in your paper, i.e., "We transfer semantic occupancy prediction to LiDAR segmentation by assigning the point label based on associated voxel label, and then evaluate the model on the mIoU metric.", did you assign the labels of points by its located voxel directly? Or did you use some interpolation strategies? Such as, trilinear interpolation on the voxel logits like OccFormer.

Besides, did you train the networks again with sparse labeled voxels as supervision signal, which is generated by the sparse lidar segmentation ground truth?

**Comment by YuanxianH** [comment]
BTW, as shown in Table 15 of the paper, you present the performance of the lidar segmentation task with 0.25m voxel_size. However, the resolution of the GT voxel you provided is 0.5m. How did you get a smaller resolution?

**Comment by tongwwt** [comment]
For the lidar segmentation task, we assign the labels of points by its located voxel directly. We generated the occupancy dataset with different resolution including 0.25m, 0.5m and 1m as shown in table 15.

**Comment by YuanxianH** [comment]
> For the lidar segmentation task, we assign the labels of points by its located voxel directly. We generated the occupancy dataset with different resolution including 0.25m, 0.5m and 1m as shown in table 15.

Thank you for getting back to me. What is the resolution setting for the experiment in Table 4? We reproduced the OccNet(r101) by your provided config and assigned the label of points directly. However, the lidar segmentation mIoU in nus validation was lower than the results in the paper.

**Comment by tongwwt** [comment]
Thanks for your interest in our work. The resolution setting in Table 4 is 0.5m.

**Comment by YuanxianH** [comment]
Well, would you mind provide weights of occnet-tiny and occnet-base? We reproduced the lidar segmentation task as your description. The lidar segmentation mIoU is a little lower, although the voxel mIoU is approximately the same.
